samples/video/capture/boards/mm_swiftio.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_VIDEO_OV7725=y
+CONFIG_I2C=y
```

Comment:
Please add this board to `samples/video/capture/sample.yaml` to make sure we build it in CI

Reply:
I didn’t find any documentation about sample.yaml. I added one by referring to the existing example. It doesn’t seem to cause CI check fail, but I’m not sure that CI checked mm_swiftio or not. Please help to review whether this modification is correct. Thanks so much!